indico/modules/events/tracks/blueprint.py

```diff
@@ -16,13 +16,17 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.events.tracks.controllers import RHManageTracks, RHCreateTrack, RHCreateTrackOld
+from indico.modules.events.tracks.controllers import (RHManageTracks, RHCreateTrack, RHCreateTrackOld, RHEditTrack,
+                                                      RHSortTracks, RHDeleteTrack)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('tracks', __name__, template_folder='templates', virtual_template_folder='events/tracks',
-                      url_prefix='/event/<confId>')
+                      url_prefix='/event/<confId>/manage')
 
-_bp.add_url_rule('/manage/tracks/', 'manage', RHManageTracks)
-_bp.add_url_rule('/manage/tracks/create', 'create', RHCreateTrack, methods=('GET', 'POST'))
+_bp.add_url_rule('/tracks/', 'manage', RHManageTracks)
+_bp.add_url_rule('/tracks/create', 'create', RHCreateTrack, methods=('GET', 'POST'))
+_bp.add_url_rule('/tracks/<int:id>/edit', 'edit_track', RHEditTrack, methods=('GET', 'POST'))
```

Comment:
could be just `/tracks/<int:track_id>` without `/edit` unless we add more stuff "within" `/tracks/<int:track_id>/...` later